apps/src/utils.js

```diff
@@ -237,6 +237,26 @@ if (!String.prototype.repeat) {
 }
 
 /**
+ * A low-performance polyfill for toBlob based on toDataURL. Adapted from:
+ * https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob
+ */
+if (!HTMLCanvasElement.prototype.toBlob) {
+  Object.defineProperty(HTMLCanvasElement.prototype, 'toBlob', {
+    value: function (callback, type, quality) {
+      const binStr = atob(this.toDataURL(type, quality).split(',')[1]);
+      const arr = new Uint8Array(binStr.length);
+
+      for (let i = 0; i < binStr.length; i++) {
+        arr[i] = binStr.charCodeAt(i);
+      }
+
+      const blob = new Blob([arr], {type: type || 'image/png'});
+      callback(blob);
+    }
+  });
+}
+
+/**
```

Comment:
Makes sense to put this next to our other polyfill, but wonder if all of our polyfills should be in a separate file from src/utils?